lib/swing-lib/src/main/java/org/triplea/swing/key/binding/KeyCode.java

```diff
@@ -19,6 +19,8 @@
   COMMA(KeyEvent.VK_COMMA),
   EQUALS(KeyEvent.VK_EQUALS),
   MINUS(KeyEvent.VK_MINUS),
+  ADD(KeyEvent.VK_ADD),
+  SUBTRACT(KeyEvent.VK_SUBTRACT),
```

Comment:
SUBTRACT and MINUS are essentially synonyms. A javadoc on this entry to mention that this is the keypad only would help clarify that difference for future maintainers. Would you mind adding such a note? Similar for 'ADD'.